### PR TITLE
NEXUS-8725: Privilege and Role descriptions

### PR DESCRIPTION
**Pull request by cstamas**
https://issues.sonatype.org/browse/NEXUS-8725

CI
http://bamboo.s/browse/NX3-OSSF423

### CODE REVIEW

**Comment by jdillon** [comment]
+1 looks reasonable

File: plugins/security/nexus-ldap-plugin/src/main/java/org/sonatype/nexus/ldap/internal/StaticSecurityConfigurationResourceImpl.groovy [hunk 1]
```diff
@@ -67,6 +71,7 @@ class StaticSecurityConfigurationResourceImpl
             ),
             new CPrivilege(
                 id: 'ldap-delete',
+                description: 'Delete permission for Ldap',
```
Comment (joedragons) [inline plugins/security/nexus-ldap-plugin/src/main/java/org/sonatype/nexus/ldap/internal/StaticSecurityConfigurationResourceImpl.groovy]: As an acronym should these be "LDAP"?  I am not sure if that's consistant across the site but I feel like it is mostly.

**Comment by adreghiciu** [comment]
+1 What about description of dynamically generated permissions?

**Comment by cstamas** [comment]
413e2ab48069af0ebab2d4caa7df1983cc667226

**Comment by jdillon** [comment]
ugh

-1 the dynamic permission descriptions added are not proper

**Pull request closed without merging**